app/src/main/java/com/sopt/now/LoginActivity.kt

```diff
@@ -0,0 +1,48 @@
+package com.sopt.now
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import com.sopt.now.databinding.ActivityLoginBinding
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding : ActivityLoginBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+
+        // 인텐트에서 데이터 추출
+        val getId = intent.getStringExtra("id")
+        val getPw = intent.getStringExtra("pw")
+        val getName = intent.getStringExtra("name")
+        val getMbti = intent.getStringExtra("mbti")
+        Log.d("login","${getId}, ${getPw}, ${getName}, ${getMbti}")
```

Comment:
```suggestion
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityLoginBinding.inflate(layoutInflater)
        
         getUserData()
         setUserData()
```
위의 예시처럼 `onCreate()` 안에는 함수화하는 것을 권장합니다!!
그래야 어떤 기능을 하는지 파악하기에도 용이하고 유지보수성이 높아질 것 같네용
제 코드 참고하셔도 좋을 것 같아요!